assets/js/test/algolia-embedded-search_test.js

```diff
@@ -33,8 +33,8 @@ describe("AlgoliaEmbeddedSearch", () => {
     window.jQuery = jsdom.rerequire("jquery");
     window.autocomplete = jsdom.rerequire("autocomplete.js");
     window.encodeURIComponent = str => str;
-    window.Turbolinks = {
-      visit: sinon.spy()
+    window.location = {
+      assign: sinon.spy()
     };
```

Comment:
Turns out JS can't simply just set `window.location` like that. But this worked to fix this test file for me, and making a similar change in the other affected places might fix the other tests as well.

```suggestion
    Object.defineProperty(window, "location", {
      value: {
        assign: sinon.spy()
      }
    });
```